CHANGELOG.md

```diff
@@ -7,26 +7,30 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ## Unreleased
 
-## Bugfix
+### New feature
+
+## 1.2.1 - 2020-12-16
+
+### Bugfix
 
 - Addressed part of issue https://github.com/inrupt/solid-client-authn-js/issues/684,
 by providing a `browser` entry in the `package.json` file. The ES modules export will
 be adressed in a different PR. 
 - The WebID is now set on the session when logging in a script.
+- When logging in with a refresh token (e.g. for a script), if the provided credentials are incorrect, an error is thrown.
 
-## New feature
```

Comment:
To consider: adding a line before and after unreleased changes will prevent merges from mangling this. (In solid-client I added a line like "The following changes are not released yet", but something inside `<!-- an HTML comment ->` would work as well.

Not for this, just an idea for later :)